etl/steps/data/grapher/artificial_intelligence/2024-03-22/papers_with_code_coding.py

```diff
@@ -0,0 +1,37 @@
+"""Load a garden dataset and create a grapher dataset."""
```

Comment:
Similarly here - you could have `shared.py` with method

```
def run(dest_dir: str, dataset_name: str):
  ...
```
and then call from every dataset
```
from . import shared
run = shared.run("papers_with_code_coding")
```

Reply:
yeaah sorry I was being lazy :D but done it now!